src/helper/FetchName.js

```diff
@@ -0,0 +1,13 @@
+const FetchName = async () => {
```

Comment:
it's not a class, just a function
```suggestion
const fetchName = async () => {
```